### PR TITLE
upgrade for react 0.15

### PR DESCRIPTION
**Pull request by revolunet**
Hi! any idea ho to handle retro-compatibility ?

### CODE REVIEW

**Comment by revolunet** [comment]
added an example in the README

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# Diaporama React bindings
+diaporama-react
+=========
+
+Use the awesome [diaporama](https://github.com/gre/diaporama) as a react component
```
Comment (gre) [inline README.md]: could you replace this to something like:

Implement a React Component to use the [diaporama](https://github.com/gre/diaporama) library.

**Comment by gre** [comment]
that's ok if it's not retro compat'. I can just make this a major release I guess.

**Comment by gre** [comment]
Thanks for this pr :)

**Comment by revolunet** [comment]
fixed!

File: README.md [hunk 2]
```diff
@@ -17,3 +21,92 @@ Use diaporama as a react component
...
+const diaporamaData = require('./diaporama.json');
+
+let props = {
+    data: diaporamaData,
+    width: 320,
+    height: 240,
+    loop: true,
+    autoplay: true,
+    onDiaporamaCreated: () => {
+      console.log('onDiaporamaCreated', arguments);
```
Comment (gre) [inline README.md]: I think arguments don't work in arrow functions. you should spread `...args`

**Comment by revolunet** [comment]
oops sorry again :)

**Comment by gre** [comment]
thanks :)

**Comment by revolunet** [comment]
👍

**Pull request merged**